tests/queries/0_stateless/01515_with_global_and_with_propagation.sql

```diff
@@ -15,3 +15,22 @@ explain syntax with 1 as x select *, x from (with 2 as x select x as y);
 explain syntax with 1 as x select x union all select x;
 explain syntax with 1 as x select x union all with 2 as x select x;
 explain syntax with 5 as q1, x as (select number + 100 as b, number as a from numbers(10) where number > q1) select * from x;
+
+-- https://github.com/ClickHouse/ClickHouse/issues/19222
```

Comment:
@Algunenano please add separate test, we prefer always add separate tests instead of modifying old ones.

Reply:
Sure, will do. Thanks